boltons/timeutils.py

```diff
@@ -11,12 +11,41 @@
 
 import re
 import bisect
+import math
+import time
 import datetime
 from datetime import timedelta
 
+# optional dependency: monotonic time
+try:
+    # python3.3 added monotonic time to stdlib
+    monotonic = time.monotonic
+except AttributeError:
+    try:
+        from importlib import import_module
+    except ImportError:
+        monotonic = None # py2.6
+    else:
+        # lib name, function that gets value given module (in order of preference)
+        _monotonic_libs = [
```

Comment:
It would be better to explicitly pass in a `monotonic` function to the `DecayCounter` instance.  Not only is the logic clearer, it also enables me to use arbitrary bindings to my platform's appropriate monotonic time interface without monkeypatching the module